scripts/mcptam_install.bash

```diff
@@ -102,7 +102,7 @@ build_mcptam()
 
 	if [ "$CI" = true ]; then
 		# grab pull request version to be tested
-		cd ~
+		cd ..
```

Comment:
This is a very careful cd .., though, as travis has us in a particular folder that it launched the script from, and we are copying that version of the repo to the newly created catkin workspace to compile.  It's possible this dir is known, but Chris was continuously jumping back to it with cd -.  Maybe we should set a variable that reads this mysterious directory, and just uses it when needed.